app/views/users/new.html.erb

```diff
@@ -0,0 +1,25 @@
+
+ <%= form_for(@user, as: :user, url: users_path, authenticity_token: form_authenticity_token)do |user|%>
+  <%= render 'shared/error_messages' %>
+  <%= user.label(:username, 'Username') %>
```

Comment:
We don't need `authenticity_token`, `as` nor `url` options for form_for.

Check [this](https://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-form_for) in the arguments section.